packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/useBraintreeSummary.js

```diff
@@ -0,0 +1,86 @@
+import { useQuery } from '@apollo/client';
+import { useTypePolicies } from '@magento/peregrine';
+import { useCartContext } from '../../../context/cart';
+
+import mergeOperations from '../../../util/shallowMerge';
+import defaultOperations, { CUSTOM_TYPES } from './braintreeSummary.gql';
+
+const mapBillingAddressData = rawBillingAddressData => {
+    if (rawBillingAddressData) {
+        const { street, country, region } = rawBillingAddressData;
+
+        return {
+            ...rawBillingAddressData,
+            street1: street[0],
+            street2: street[1],
+            country: country.code,
+            state: region.code
+        };
+    } else {
+        return {};
+    }
+};
+
+/**
+ * Talon for the braintree summary view.
+ *
+ * @param {DocumentNode} props.operations operations used by this summary component
+ *
+ * @returns {
+ *   billingAddress: {
+ *      firstName: String,
+ *      lastName: String,
+ *      country: String,
+ *      street1: String,
+ *      street2: String,
+ *      city: String,
+ *      state: String,
+ *      postalCode: String,
+ *   },
+ *   paymentNonce: {
+ *      type: String,
+ *      description: String,
+ *      details: {
+ *          cardType: String,
+ *          lastFour: String,
+ *          lastTwo: String
+ *      },
+ *   },
+ *   isBillingAddressSame: Boolean,
+ *   isLoading: Boolean,
+ * }
+ */
+export const useBraintreeSummary = (props = {}) => {
+    const { typePolicies = CUSTOM_TYPES } = props;
+    const operations = mergeOperations(defaultOperations, props.operations);
+
+    const { getBraintreeSummaryData } = operations;
+
+    useTypePolicies(typePolicies);
```

Comment:
I believe this was existing, but recall we've determined this doesn't do anything and could even cause cache corruption. If we can easily remove, let's try in this scope.

Reply:
If I understand the doc right if we remove this `paymentNonce` maybe still in cache save right ? But I am  not sure. 

https://www.apollographql.com/docs/react/caching/cache-field-behavior/

```
export const CUSTOM_TYPES = {
    Cart: {
        fields: {
            /**
             * @client field policies must be defined if queried along server
             * props or the entire query will return null.
             */
            isBillingAddressSame: {
                read(cached) {
                    return cached !== false;
                }
            },
            paymentNonce: {
                read(cached) {
                    return cached || null;
                }
            }
        }
    }
};
```

Reply:
Okay I tested we can remove without any issue